CMakeLists.txt

```diff
@@ -90,7 +90,9 @@ elseif(APPLE)
 	set(CMAKE_INSTALL_MANDIR "${CMAKE_INSTALL_PREFIX}/Resources/man/man6")
 endif()
 
-set(CMAKE_CXX_FLAGS_RelWithDebInfo "${CMAKE_CXX_FLAGS_RelWithDebInfo} -O3")
+if(NOT MSVC)
+	set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} -O3")
+endif()
```

Comment:
Let's also add optimizations for msvc. I think it was -O2 (https://learn.microsoft.com/en-us/cpp/build/reference/o-options-optimize-code?view=msvc-170) optimize for speed

Reply:
/O2 but it's already on, I believe.